src/backend/core/tests/documents/test_api_documents_cors_proxy.py

```diff
@@ -0,0 +1,156 @@
+"""Test on the CORS proxy API for documents."""
+
+import random
+
+from django.contrib.auth.models import AnonymousUser
+
+import pytest
+from rest_framework.test import APIClient
+
+from core import factories
+
+pytestmark = pytest.mark.django_db
+
+
+@pytest.fixture
+def base64_doc_content():
+    """
+    Valid document content in base64 encoding.
+    It contains an image embedded in the document.
+    This image is available at the following URL:
+    https://upload.wikimedia.org/wikipedia/commons/3/31/Wiki_logo_Nupedia.jpg
+    """
+    return (
+        "AgLAwcbuAwCo88b1ogMWAXcVV2lraV9sb2dvX051cGVkaWEuanBnqPPG9aIDFwF3SWh0dHBzOi8vdXBsb2FkLndpa"
+        "2ltZWRpYS5vcmcvd2lraXBlZGlhL2NvbW1vbnMvMy8zMS9XaWtpX2xvZ29fTnVwZWRpYS5qcGcV88b1ogMABwEOZG"
+        "9jdW1lbnQtc3RvcmUDCmJsb2NrR3JvdXAHAPPG9aIDAAMOYmxvY2tDb250YWluZXIBAPPG9aIDAQEAAygA88b1ogM"
+        "BAmlkAXcOaW5pdGlhbEJsb2NrSWQoAPPG9aIDAQl0ZXh0Q29sb3IBdwdkZWZhdWx0KADzxvWiAwEPYmFja2dyb3Vu"
+        "ZENvbG9yAXcHZGVmYXVsdIfzxvWiAwEDDmJsb2NrQ29udGFpbmVyBwDzxvWiAwkDCXBhcmFncmFwaCgA88b1ogMKD"
+        "XRleHRBbGlnbm1lbnQBdwRsZWZ0KADzxvWiAwkCaWQBdyQ1ZTU4ZmQxYi02ODI1LTQ0NmUtOWY0Yy1kZjBhYzcxZD"
+        "MxZmYoAPPG9aIDCQl0ZXh0Q29sb3IBdwdkZWZhdWx0KADzxvWiAwkPYmFja2dyb3VuZENvbG9yAXcHZGVmYXVsdAA"
+        "FR/PG9aIDAgMFaW1hZ2UoAPPG9aIDFA10ZXh0QWxpZ25tZW50AXcEbGVmdCgA88b1ogMUBG5hbWUBdwAoAPPG9aID"
+        "FAN1cmwBdwAoAPPG9aIDFAdjYXB0aW9uAXcAKADzxvWiAxQLc2hvd1ByZXZpZXcBeCgA88b1ogMUDHByZXZpZXdXa"
+        "WR0aAF9gAgB88b1ogMDAgQPBRYC"
+    )
+
+
+def test_api_docs_cors_proxy_valid_url(base64_doc_content):
+    """Test the CORS proxy API for documents with a valid URL."""
+    document = factories.DocumentFactory(link_reach="public")
+    document.content = base64_doc_content
+    document.save()
+
+    client = APIClient()
+    url_to_fetch = (
+        "https://upload.wikimedia.org/wikipedia/commons/3/31/Wiki_logo_Nupedia.jpg"
```

Comment:
It seems to be flaky and send a 403, maybe we should use a more reliable one ? 
What about that: https://docs.numerique.gouv.fr/assets/logo-gouv.png